src/asciidoc/docs/develop/testingTemplates.adoc

```diff
@@ -0,0 +1,164 @@
+= Testing templates
+:page-slug: testing-templates
+:page-description: A consolidated resource for methods a developer can use to test templates
+
+There are many ways you can test your Koji templates.
+Typically, you will begin testing in the Koji editor.
+Later, you'll want to publish the template as *Unlisted*, so you can test it in a production environment before giving access to general users.
+Some of the most common testing methods are listed below.
+
+== Testing in the Koji editor
+
+The Koji editor provides various ways to preview a template's behavior.
+These are covered in more detail in the documentation for the editor (see <<editor#>>).
+
+=== Previewing the Koji
+
+The *Live preview* tab allows you to toggle between *Preview* mode, which shows what the Koji looks like when running, and *Editing* mode, which allows you to perform remixing in the preview.
+
+The *Remote* tab offers several additional ways of previewing your Koji.
+You can test it on your mobile device by scanning a QR code.
+There are also options for opening the Koji in a new tab, running it in a live player, or copying the link for the preview.
```

Comment:
There are some awesomely powerful testing capabilities here that could use more explanation. For example, the live player is such a huge improvement over what we had before, and it enables many more testing modes (e.g. in-app purchases and live feed, which you mention later in the page but could summarize here). Let's explain the what/why of these tools more. These changes might also play into how you reorganize the sections (see other comment).